bindings/c/test/mako/mako.c

```diff
@@ -492,11 +518,17 @@ int run_one_transaction(FDBTransaction* transaction, mako_args_t* args, mako_sta
 					rc = run_op_insert(transaction, keystr, valstr);
 					if (rc == FDB_SUCCESS) {
 						/* commit insert so mutation goes to storage */
+						// timer_1
```

Comment:
Please have a little more meaningful comment here 😉

Reply:
I am sorry, forgot to remove the comment.